src/command_modules/azure-cli-servicefabric/azure/cli/command_modules/servicefabric/_params.py

```diff
@@ -4,128 +4,126 @@
-                     **enum_choice_list(['WindowsServer2012R2Datacenter', 'WindowsServer2016Datacenter', 'WindowsServer2016DatacenterwithContainers', 'UbuntuServer1604']))
+                      **enum_choice_list(['WindowsServer2012R2Datacenter',
+                                          'WindowsServer2016Datacenter',
+                                          'WindowsServer2016DatacenterwithContainers',
+                                          'UbuntuServer1604']))
 
 register_cli_argument('sf client certificate', 'certificate_common_name',
                       help='client certificate common name.')
-register_cli_argument('sf client certificate', 'admin_client_thumbprints',
-                      help='list of client certificate thumbprint that only has admin permission.')
+register_cli_argument('sf client certificate', 'admin_client_thumbprints', nargs='+',
+                      help='Space separated list of client certificate thumbprint that only has admin permission, ')
 register_cli_argument('sf client certificate', 'certificate_issuer_thumbprint',
                       help='client certificate issuer thumbprint.')
 
+register_cli_argument('sf cluster certificate', 'thumbprint',
+                      help='The cluster certificate thumbprint to be removed')
+
 register_cli_argument('sf cluster client-certificate', 'is_admin',
                       help='Client authentication type.')
 register_cli_argument('sf cluster client-certificate', 'certificate_issuer_thumbprint',
                       help='client certificate issuer thumbprint.')
 register_cli_argument('sf cluster client-certificate', 'certificate_common_name',
                       help='client certificate common name.')
-register_cli_argument('sf cluster client-certificate', 'admin_client_thumbprints',
+register_cli_argument('sf cluster client-certificate', 'admin_client_thumbprints', nargs='+',
                       help='client certificate thumbprint that only has admin permission.')
-register_cli_argument('sf cluster client-certificate', 'readonly_client_thumbprints',
-                      help='list of client certificate thumbprint that has read only permission.') 
+register_cli_argument('sf cluster client-certificate', 'readonly_client_thumbprints', nargs='+',
+                      help='Space separated list of client certificate thumbprint that has read only permission.')
 register_cli_argument('sf cluster client-certificate add', 'thumbprint',
                       help='client certificate thumbprint.')
-register_cli_argument('sf cluster client-certificate remove', 'thumbprint',
-                      help='a single or list of client certificate thumbprint(s) to be remove.')
+register_cli_argument('sf cluster client-certificate remove', 'thumbprints', nargs='+',
+                      help='A single or Space separated list of client certificate thumbprint(s) to be remove.')
```

Comment:
You can just say "Space separated list of ...". That's the common pattern in the CLI and no one has ever complained that the didn't realize they could use just one. :)